adapters/syslog/syslog.go

```diff
@@ -29,6 +30,15 @@ func getopt(name, dfault string) string {
 	return value
 }
 
+func tplGetEnvVar(env []string, key string) string {
+        for _, value := range env {
+                if strings.HasPrefix(value, fmt.Sprintf("%s%s", key, "="))  {
```

Comment:
Maybe do the `fmt.Sprintf("%s%s", key, "=")` once in the beginning of the function? Saves a few allocs.